R/double_ml_pliv.R

```diff
@@ -153,10 +168,10 @@ DoubleMLPLIV = R6Class("DoubleMLPLIV",
     #' Number of repetitions for the sample splitting. Default is `1`.
     #'
     #' @param score (`character(1)`, `function()`) \cr
-    #' A `character(1)` (`"partialling out"` is the only choice) or a
-    #' `function()` specifying the score function.
+    #' A `character(1)` (`"partialling out"` or `IV-type`) or a `function()`
```

Comment:
Consistent format if `score` is a `character`

```suggestion
    #' A `character(1)` (`"partialling out"` or `"IV-type"`) or a `function()`
```

Reply:
Done